packages/app-store/routing-forms/playwright/tests/basic.e2e.ts

```diff
@@ -93,20 +93,20 @@ test.describe("Routing Forms", () => {
       await gotoRoutingLink({ page, formId });
       await expect(page.locator("text=Test Form Name")).toBeVisible();
 
-      await page.goto(`apps/routing-forms/route-builder/${formId}`);
+      await page.goto(`routing/route-builder/${formId}`);
```

Comment:
do you think it's too much to keep the tests for the old paths to ensure they're still redirected correctly in the future? (in case someone makes some change unknowingly)

Reply:
We can do that, will do!